clang/lib/Sema/SemaRISCVVectorLookup.cpp

```diff
@@ -282,6 +282,9 @@ void RISCVIntrinsicManagerImpl::ConstructRVVIntrinsics(
         }
       }
 
+      if (BaseType == BasicType::BFloat16 && !TI.hasFeature("zvfbfmin"))
```

Comment:
bfloat vector is only valid when zvfbfmin is enabled. So it doesn't need to check again.